css-positioning/absolute.css

```diff
@@ -0,0 +1,54 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+  top: 0;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  top: 16px;
+  right: 20px;
+  z-index: 2;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+
+  position: absolute;
+  bottom: 5px;
+  left: 10px;
+  z-index: 3;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+
+  /* your code here */
+
+  position: absolute;
+  height: 300px;
+  width: 300px;
+  top: 0;
+  left: 0;
+  z-index: 1;
```

Comment:
You're setting this to have a width and height of 300px. What happens if your image were larger than 300px? You'd have to come over here and modify this again. We should be able to have this cover the size of it's relative ancestor by just using the positioning properties (top, right, bottom, and left)